docs/UserGuide.md

```diff
@@ -245,22 +351,59 @@ _Details coming soon ..._
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Known issues
+## Troubleshooting
+
+Facing issues with SweetRewards? Here are some common problems and how to resolve them.
 
-1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
+### Technical Issues
+1. Application doesn't start
+   * 🚫 Problem: Double-clicking the jar file doesn’t open SweetRewards. 
+   * ✅ Solution: Ensure Java 11 or above is installed on your machine. Open a command terminal, navigate to the folder containing the jar file, and run java -jar sweetrewards-v1.2.jar. If the problem persists, verify that the downloaded jar file is not corrupted and re-download if necessary.
 
+2. GUI opens off-screen
+
+   * 🚫 Problem: When using multiple screens, SweetRewards opens off the visible screen. 
+   * ✅ Solution: Delete the preferences.json file found in the same folder as the jar file. Restart SweetRewards.
+
+### Operational Questions
+1. Adding Points to Wrong Member
+    * 🚫 Problem: Points were mistakenly added to the wrong member. 
+    * ✅ Solution: Use the edit command to adjust the points for both involved members. Subtract the incorrectly added points from the wrong member and add them to the correct one. For example, edit 2 p/-50 to remove points and edit 3 p/50 to add them where needed.
```

Comment:
I think the edit command wasn't implemented like this? I gave it a try with edit 1 p/-50 and it didn't work

Reply:
Yeah it just directly replaces the old points with the new ones

Reply:
Thank you for pointing it out!